lib/cmake/thrust/README.md

```diff
@@ -38,6 +38,22 @@ If using Thrust from the CCCL sources, this would be
 $ cmake . -DThrust_DIR=<CCCL git repo root>/thrust/thrust/cmake/
 ```
 
+#### Large Array (64-bit offseet) Handling: `DISPATCH`
+
+The `DISPATCH` option allows users to select the tradeoff of compile-time / binary-size
+vs. performance vs. scalability when given large inputs that require 64-bit offset types.
+This currently only applies when DEVICE=CUDA.
+
+- `Dynamic` Compiles each kernel twice, once for 32-bit offsets and again for 64-bit offsets,
+  and chooses dynamically using the input size at runtime.
```

Comment:
Suggestion: IIUC, this is not always true. Some algorithms switched to int64 regardless and others just chunk the input into 32-bit addressable pieces. Maybe rephrase to `May compile each kernel twice`.